Assets/MixedRealityToolkit-SDK/Features/Boundary/System/MixedRealityBoundarySystem.cs

```diff
@@ -178,26 +256,36 @@ public int GetHashCode(object obj)
 
         #region IMixedRealityBoundarySystem Implementation
 
+        /// <summary>
+        /// The thickness of three dimensional generated boundary objects.
+        /// </summary>
+        private const float boundaryObjectThickness = 0.005f;
+
+        /// <summary>
+        /// A small offset to avoid render conflicts, primarilly with the floor.
+        /// </summary>
+        /// <remarks>
+        /// This offset is used to avoid consuming mutiple physics layers.
+        /// </remarks>
+        private const float boundaryObjectRenderOffset = 0.001f;
+
+        /// <summary>
+        /// Parent <see cref="GameObject"/> which will encapsulate all of the teleportable boundary visualizations.
+        /// </summary>
+        private GameObject boundaryVisualizationParent;
+
+        /// <summary>
+        /// Layer used to tell the (non-floor) boundary objects to not accept raycasts
+        /// </summary>
+        private readonly int ignoreRaycastLayer = LayerMask.NameToLayer("Ignore Raycast");
```

Comment:
if we know the layer index, why not just assign it here since it's private? 

```
/// <summary>
/// Layer used to tell the (non-floor) boundary objects to not accept raycasts
/// </summary>
/// <remarks>
/// Default is Ignore Raycast layer.
/// </remarks>
 private readonly int ignoreRaycastLayer = 2; 
```